spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -1,4 +1,10 @@
 describe ManageIQ::Providers::AnsibleTower::AutomationManager::Refresher do
+  let(:more_credential_types) do
+    {
+      'hello_sat_cred' => 'Satellite6Credential'
```

Comment:
Is this expected to be a setting that is configured in `settings.yml`?

If so, add a comment to the settings file in this repo to show that what setting would look like if someone wanted to override it.

Reply:
Greg, this is not a setting... and hmm... I'm not sure you ask this way...

This refresher test is based on a cassette which contains of all types of credential types. This used to be in for both embedded and external one. I'm pulling this out here so that embedded won't fail.

Reply:
K, I was actually confused by `hello_sat_cred`.  I thought that was some dummy name.  But I found it in the actual code for defining the Sat6 Credential.

I understand how this is being used now.